### PR TITLE
 #158834383. Have all API V2 Endpoints working.

### PR DESCRIPTION
**Pull request by Mutuba**
### What does the PR do?

Implements the following endpoints:
- POST/api/v2/auth/register
- POST/api/v2/auth/login
- POST/api/v2/rides
- GET/api/v2/rides
- GET/api/v2/rides/ride_id
- PUT/api/v2/rides/ride_id
- DELETE/api/v2/rides/ride_id
- POST/api/v2/rides/ride_id/requests
- GET/api/v2/rides/request_id
- DELETE/api/v2/rides/request_id
- PUT/api/v1/rides/request_id

## How to manually test
- Clone the repo from github. Change directory into Ride-My-Way3- and run python run.py , to start the server.
- Open postman and test all the above endpoints to see the results.
- Remember after registering, login gives you an access token to authenticate you for resources that need token. In Post Man: enter access-token then provide the token generated from login endpoint.

## Relevant pivotal tracker stories
- #158834383

### CODE REVIEW

File: instance/config.py [hunk 1]
```diff
@@ -0,0 +1,43 @@
+# config file
+
+import os
+
+SECRET_KEY = "Mysecret"
```
Comment (Gidraff) [inline instance/config.py:5]: Hey, allowing important credentials such as  `secret_key` to a public repo is not considered a good practice. Such data should be abstracted and by that I mean, you can export(set) them when starting your application and read them from your environment variables using `os.getenv()`.

File: run.py [hunk 2]
```diff
@@ -0,0 +1,15 @@
+# Application entry point
+
+from app import create_app
+from flask_cors import CORS
+import os
+import psycopg2
+from create_tables import create_tables
+
+app = create_app("development")
```
Comment (Gidraff) [inline run.py:9]: Hey, you should abstract this as well.

**Review by Gidraff — COMMENTED** [review]
Your code is clean and self-documenting. However, you need to make your test cases more robust. You are only asserting whether the status code returned is what you expected. An example of a good unit test would be, testing whether the size of a database table increases when data is inserted.

File: run.py (hunk 2 above)
Comment (Mutuba) [reply run.py:9]: I appreciate, working on it

**Pull request merged**